src/view/ui.py

```diff
@@ -107,34 +170,47 @@ def __init__(self, parent):
                           QtWidgets.QGraphicsItem.ItemSendsScenePositionChanges)
 
         def itemChange(self, change, value):
+            """
+            send position changed signal and update current position
+            """
             if change in [QtWidgets.QGraphicsItem.ItemPositionChange,
                           QtWidgets.QGraphicsItem.ItemVisibleChange]:
                 self.parent.positionChanged.emit()
-                self.parent.current_position = self.parent.handle.pos()
+                self.parent.currentPosition = self.parent.handle.pos()
             return QtWidgets.QGraphicsRectItem.itemChange(self, change, value)
 
         def mousePressEvent(self, event=None):
+            """
+            set selected if mouse clicked
+            """
```

Comment:
In my opinion, those docstrings are not really relevant, the method name seems sufficient. Maybe only add docstring when necessary, when the function has some inputs.
What do you think?

Reply:
yep good idea